templates/partials/header.html

```diff
@@ -170,4 +170,5 @@
             <link rel="stylesheet" href="{{ get_url(path=stylesheet) }}">
         {% endfor %}
     {% endif %}
+    <script src="{{config.base_url | safe }}js/mermaid.js" nonce="{{config.extra.nonce}}"></script>
```

Comment:
Thanks for the PR! What's the use of the nonce here? Can it be removed or hardcoded instead of specifying it in the config.toml?

Reply:
my assumption was that it was necessary for CSP, but turns out it is not, have removed!